src/App.js

```diff
@@ -44,15 +47,9 @@ export default class App extends Component {
     setCurrentPropId: (currentPropId) => {
       this.setState({ currentPropId });
     },
-    setShowGenderColor: (showGenderColor) => {
-      this.setState({ showGenderColor });
-    },
     setCurrentTheme: (currentTheme) => {
```

Comment:
this should also go in settings

Reply:
of course

Reply:
`currentTheme` in the settings object, so it will be saved in localstorage